backend/app/api/v1/endpoints/quay/quayJobs.py

```diff
@@ -88,7 +88,7 @@ async def jobs(
     "/api/v1/quay/filters",
     summary="Returns a filter list",
     description="Returns the data to build filters in the specified dates. \
-            If not dates are provided the API will default the values. \
+            If not dates are provided the API will use the following values as defaults. \
```

Comment:
"If dates are not provided"

Reply:
It may also be clearer if it ended in a colon :.